types/draft-js/index.d.ts

```diff
@@ -235,6 +234,8 @@ declare namespace Draft {
 
                 onBlur?(e: SyntheticEvent): void;
                 onFocus?(e: SyntheticEvent): void;
+                onCopy?(editor: Editor, e: ClipboardEvent): void;
+                onCut?(editor: Editor, e: ClipboardEvent): void;
```

Comment:
I feel that it would be good to be more explicit with the event typing here. `ClipboardEvent` references a React synthetic clipboard event, so should probably be defined like this around line 24:25:
```typescript
type SyntheticClipboardEvent = React.ClipboardEvent<{}>;
```

And then:
```suggestion
                onCopy?(editor: Editor, e: SyntheticClipboardEvent): void;
                onCut?(editor: Editor, e: SyntheticClipboardEvent): void;
```

Reply:
@PeterDekkers You are right, I have changed according to your suggestion 
5e07e8b